src/sentry/integrations/__init__.py

```diff
@@ -32,16 +32,11 @@
 )
 
 from .analytics import register_analytics
```

Comment:
this file contains the main changes, rest are modifications of imports b/c of this change.